app/controllers/admin/organizations_controller.rb

```diff
@@ -83,19 +79,23 @@ def destroy
   private
 
   def organization_params
-    params.require(:organization).permit(
+    # Not sure why params.permit wasn't honoring this field for the native simple_form array
```

Comment:
Since it's an array you need to do `person_ids: []`

https://github.com/rails/strong_parameters#permitted-scalar-values